.travis.yml

```diff
@@ -1,12 +1,22 @@
 sudo: false
+os:
+  - linux
+  - osx
 language: node_js
 node_js:
+  - node
   - '6'
   - '5'
-  - '4'
```

Comment:
yes this makes sense. I have updated the templates that generate/update this file. thanks for the reminder